airflow/models/mappedoperator.py

```diff
@@ -484,14 +486,16 @@ def _get_unmap_kwargs(self) -> Dict[str, Any]:
             **self.mapped_kwargs,
         }
 
-    def unmap(self) -> "BaseOperator":
+    def unmap(self, rendered_kwargs: Optional[Dict[str, Any]] = None) -> "BaseOperator":
```

Comment:
Probably should document `rendered_kwargs` (and maybe it’s better to call it `unmap_kwargs` for consistency?)

Reply:
Done